pkg/chartutil/chartfile.go

```diff
@@ -58,3 +62,37 @@ func SaveChartfile(filename string, cf *chart.Metadata) error {
 	}
 	return ioutil.WriteFile(filename, out, 0644)
 }
+
+// IsChartDir validate a chart directory.
+//
+// Checks for a valid Chart.yaml.
+func IsChartDir(dirName string) (bool, error) {
+	if fi, err := os.Stat(dirName); err != nil {
+		return false, err
+	} else if !fi.IsDir() {
+		return false, fmt.Errorf("%q is not a directory", dirName)
+	}
+
+	chartYaml := filepath.Join(dirName, "Chart.yaml")
```

Comment:
the issue why i was validating yaml here was because LoadDir scans all the directories in the requested directory, so tried to minimize the chances of scanning a directory in the absence of a valid Chart.yaml